web3.md

```diff
@@ -741,6 +750,55 @@ eth_sendRawTransaction
     rule <k> #eth_sendRawTransactionSend TXID => #sendResponse( "result": "0x" +String #hashSignedTx( TXID ) ) ... </k>
 ```
 
+Transaction Receipts
+--------------------
+- The transaction receipt is a tuple of four items comprising: 
+  - the cumulative gas used in the block containing the transaction receipt as of immediately after the transaction has happened
+  - the set of logs created through execution of the transaction
+  - the Bloom filter composed from information in those logs
+  - the status code of the transaction.
+
+```k
+    syntax KItem ::= "#makeTxReceipt" Int
+ // -------------------------------------
+    rule <k> #makeTxReceipt TXID => . ... </k>
+         <txReceipts>
+           ( .Bag
+          => <txReceipt>
+               <txHash> "0x" +String #hashSignedTx (TXID) </txHash>
+               <cumulativeGas> GLIMIT </cumulativeGas>
+               <logSet> LOGS </logSet>
+               <bloomFilter> LOGSBLOOM </bloomFilter>
+               <txStatus> 1 </txStatus>
+             </txReceipt>
+           )
+           ...
+         </txReceipts>
+         <statusCode> EVMC_SUCCESS </statusCode>
+         <logsBloom> LOGSBLOOM </logsBloom>
+         <gas> GLIMIT </gas>
```

Comment:
We should probably be using `<gasUsed>` here instead of `<gas>`.

It's being updated each transaction anyway (double-check that `#executeTx` actually does this).